src/components/OSCALMetadata.js

```diff
@@ -38,17 +54,218 @@ const OSCALMetadataPartiesCard = styled(Paper)(({ theme }) => ({
+    <Typography>
+      <Link href={`mailto:${props.email}`}>{props.email}</Link>
+    </Typography>
+  );
+}
+
+export function OSCALMetadataPartyTelephone(props) {
+  const { telephone } = props;
+  return (
+    <Typography>
+      {getPhoneIcon(telephone.type)}
+      <Link href={`tel:${telephone.number}`}>{telephone.number}</Link>
+    </Typography>
+  );
+}
+
+export function OSCALMetadataPartyDialog(props) {
+  const [open, setOpen] = React.useState(false);
+
+  const handleOpen = () => {
+    setOpen(true);
+  };
+
+  const handleClose = () => {
+    setOpen(false);
+  };
+
+  const PartyInfoTypes = {
+    address: "address",
+    telephone: "telephone",
+    email: "email",
+  };
+
+  const TYPE_MAPPING = (infoProps) => ({
+    address: <OSCALMetadataPartyAddress address={infoProps} />,
+    telephone: <OSCALMetadataPartyTelephone telephone={infoProps} />,
+    email: <OSCALMetadataPartyEmail email={infoProps} />,
+  });
+
+  const getPartyInfoList = (
+    list,
+    partyInfoType,
+    emptyMessage = "No information provided"
+  ) => {
+    if (!list?.length) {
+      return <Typography> {emptyMessage} </Typography>;
+    }
+
+    return list.map((item) => (
+      <ListItem
+        key={partyInfoType === "email" ? item : `${item?.type}--${item?.name}`}
+      >
+        {TYPE_MAPPING(item)[partyInfoType]}
+      </ListItem>
+    ));
+  };
+
+  return (
+    <CardActions>
+      <Button size="small" variant="outlined" onClick={handleOpen}>
+        <ContactPageIcon />
+        Contact
+      </Button>
+      <Dialog
+        open={open}
+        onClose={handleClose}
+        scroll="paper"
+        aria-labelledby="scroll-dialog-title"
+        aria-describedby="scroll-dialog-description"
+        maxWidth="md"
+        fullWidth
+      >
+        <DialogTitle id="scroll-dialog-title">{props.party.name}</DialogTitle>
+        <DialogContent>{props.partyRolesText}</DialogContent>
+        <DialogContent>
+          <Grid container spacing={1}>
+            <Grid item xs={4}>
+              <Typography variant="h6" component="h3">
+                <MapIcon />
+                Addresses
+              </Typography>
+              <List>
+                {getPartyInfoList(
+                  props.party.addresses,
+                  PartyInfoTypes.address,
+                  "No address information provided"
+                )}
+              </List>
+            </Grid>
+
+            <Grid item xs={4}>
+              <Typography variant="h6" component="h3">
+                <PhoneIcon />
+                Phone
+              </Typography>
+              <List>
+                {getPartyInfoList(
+                  props.party["telephone-numbers"],
+                  PartyInfoTypes.telephone,
+                  "No telephone information provided"
+                )}
+              </List>
+            </Grid>
+            <Grid item xs={4}>
+              <Typography variant="h6" component="h3">
+                <AlternateEmailIcon />
```

Comment:
I think this is more typical for an email icon - [Material Icons - Material UI (mui.com)](https://mui.com/material-ui/material-icons/?query=email&selected=Email) instead of an @ symbol.

Reply:
The suggested icon was adopted